recipes/tmpdir/codemod.yml

```
@@ -0,0 +1,21 @@
+schema_version: "1.0"
+name: nodejs/tmpdir
```

It needs the `@` :) And the name is usually $old-to-$new

```suggestion
name: @nodejs/tmpDir-to-tmpdir
```

---

Do you mind if I try? (If successful, I'd push up a commit with it)